src/web-lock.test.ts

```diff
@@ -4,8 +4,13 @@ import {newTab, Tab} from './test-util';
 
 let tab: Tab;
 
+function isFirefox(): boolean {
+  return navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
```

Comment:
Use a regxp?

```suggestion
  return /firefox/i.test(navigator.userAgent);
```

Reply:
Good call. I definitely didn't use StackOverflow for this. ;)